evap/staff/templates/staff_evaluation_operation.html

```diff
@@ -55,9 +56,39 @@
         {% endif %}
         <input type="hidden" name="target_state" value="{{ target_state }}" />
 
+        {% if show_delete_answers_checkbox %}
+            <div class="card mb-3">
+                <div class="card-body">
+                    <div class="row">
+                        <div class="col">
+                            <div class="form-check">
+                                <input class="form-check-input" type="checkbox" name="delete-previous-answers"
+                                       id="delete-previous-answers" checked>
+                                <label class="form-check-label" for="delete-previous-answers">
+                                    {% trans "Delete previous answers" %}
+                                </label>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        {% endif %}
+
         <div class="card card-submit-area card-submit-area-2 text-center mb-3">
             <div class="card-body">
-                <button type="submit" class="btn btn-primary">{% translate 'Confirm' %}</button>
+                <!-- TODO: only show modal if show_delete_answers_checkbox and #delete-previous-answers is set -->
+                <confirmation-modal type="submit" form="evaluation-operation-form">
+                    <span slot="title">{% translate 'Reset Evaluation' %}</span>
+                    <span slot="action-text">{% translate 'Reset Evaluation' %}</span>
+                    <!-- TODO: Brainstorm better modal-message -->
+                    <span slot="question">
+                        {% blocktranslate trimmed %}
+                            This will delete all previously received answers.
+                        {% endblocktranslate %}
+                    </span>
+                    <!-- TODO: when should one use type="button" vs type="submit" while using the confirmation-model dialogue? -->
```

Comment:
The show-button should probably never be a submit button, because form elements defined here (in the users of the confirmation modal) will bind to a form outside of the modal, not within it. The show button should not submit any forms and therefore not be a submit type.

However, it may be practical to overwrite the submit row with buttons that belong to forms outside of the modal, so there it could make sense.